src/index.d.ts

```diff
@@ -259,6 +259,7 @@ declare namespace CentralServicesShared {
     ORACLE = 'oracle',
     POSITION = 'position',
     PREPARE = 'prepare',
+    PREPARE_FORWARDED = 'prepare-forwarded',
```

Comment:
Doesn't it need to be in action Enum?

Reply:
Confused. It's declared in the Event.Action part of the dict.

Reply:
Oh nvm, index.d.ts file. done.